src/main/java/sirius/web/controller/AutocompleteHelper.java

```diff
@@ -137,20 +118,39 @@ public Completion setDescription(String description) {
          *
          * @param disabled <tt>true</tt> if the suggestion is disabled, <tt>false</tt> otherwise
          * @return the completion itself for fluent method calls
+         * @deprecated Use {@link #markDisabled()}.
          */
+        @Deprecated
         public Completion setDisabled(boolean disabled) {
             this.disabled = disabled;
             return this;
         }
 
+        /**
+         * Marks the suggestion as disabled.
+         *
+         * @return the completion itself for fluent method calls
+         */
+        public Completion markDisabled() {
+            this.disabled = true;
+            return this;
+        }
+
         private void writeTo(StructuredOutput out) {
             out.beginObject("completion");
             {
-                out.property("id", value);
-                out.property("text", label == null ? "" : label);
-                if (Strings.isFilled(description)) {
-                    out.property("description", description);
+                out.property("value", value);
+                out.property("fieldLabel", fieldLabel);
+                out.property("completionLabel", Strings.isFilled(completionLabel) ? completionLabel : fieldLabel);
+                out.property("completionDescription", completionDescription);
+
+                // LEGACY SUPPORT....
+                out.property("text", fieldLabel == null ? "" : fieldLabel);
+                if (Strings.isFilled(completionDescription)) {
+                    out.property("description", completionDescription);
                 }
```

Comment:
shouldn this output completionLabel as 'description'?

Reply:
Dann müsste oben im legacy Konstruktor aber auch completionLabel statt completionDescription initialsiert werden